src/test/java/definitions/TriviaShwetaStepdefs.java

```diff
@@ -0,0 +1,248 @@
+    }
+
+    @Then("I delete the question")
+    public void iDeleteTheQuestion() {
+        getDriver().findElement(By.xpath("//span[contains(text(), 'Delete Question')]")).click();
+        //span[contains(text(), 'Delete Question')]
+    }
+
+    @Then("I add total {int} options in by clicking {string} button")
+    public void iAddTotalOptionsInByClickingButton(int numOfOptions, String addButton) throws InterruptedException {
+
+        int i=1;
+        String opt = "Option"+i;
+        for( i=1; i<numOfOptions; i++) {
+            getDriver().findElement(By.xpath("//span[contains(text(),'" + addButton + "')]")).click();
+            getDriver().findElement(By.xpath("//textarea[@placeholder='Option "+ (i+2)+"*']")).sendKeys(opt);
+            Thread.sleep(3);
+        }
+    }
+
+    @Then("I should be able to {string} the quiz with multiple choice question with fifteen options")
+    public void iShouldBeAbleToTheQuizWithMultipleChoiceQuestionWithFifteenOptions(String save) {
+        getDriver().findElement(By.xpath("//span[contains(text(),'"+save+"')]")).click();
+    }
+
+    @Then("I select {string} as correct choices")
+    public void iSelectAsCorrectChoices(String optionOne) {
+        getDriver().findElement(By.xpath("(//mat-checkbox/label/div)[2]")).click();
+
+    }
+
+    @Then("I should be able to {string} the quiz with minimum two choices")
+    public void iShouldBeAbleToTheQuizWithMinimumTwoChoices(String button) {
+        getDriver().findElement(By.xpath("//span[contains(text(),'"+button+"')]")).click();
+
+    }
+
+    @Then("setting dialogue box should have opacity zero")
+    public void settingDialogueBoxShouldHaveOpacityZero() {
+        int number = (getDriver().findElements(By.xpath("//div[style='opacity: 0; transform: scale(1, 1);']"))).size();
+        assertThat(number==0);
+    }
+
+    @Then("I add a space {string} at the beginning of question text")
+    public void iAddASpaceAtTheBeginningOfQuestionText(String space) {
+        getDriver().findElement(By.xpath("//textarea[@formcontrolname='question']")).sendKeys(space);
+    }
+
+//    @Then("{string} message should be displayed")
+//    public void messageShouldBeDisplayed(String expectedUrl) {
+//        String actualUrl = getDriver().getCurrentUrl();
+//        assertThat(actualUrl.contains(expectedUrl));
+//        System.out.println( "expectedUrl : " + expectedUrl);
+//        System.out.println( "actualUrl : " + actualUrl);
+//    }
+
+    @Then("quiz should not get saved")
+    public void quizShouldNotGetSaved() {
+        String actualUrl = getDriver().getCurrentUrl();
+        assertThat(actualUrl.contains("/quiz-builder/"));
+
+    }
+
+    @Then("I should still be at {string} page")
+    public void iShouldStillBeAtPage(String expectedUrl) {
+        String actualUrl = getDriver().getCurrentUrl();
+        System.out.println( "actualUrl : " + actualUrl);
+        Assert.assertTrue(actualUrl.contains("expectedUrl"));
```

Comment:
Assert.assertTrue(actualUrl.contains("expectedUrl"));
You should use variable name here
Assert.assertTrue(actualUrl.contains(expectedUrl));

Reply:
Oh, by mistake I think it happened.